medical_patient_disease/views/medical_patient_disease_view.xml

```diff
@@ -162,7 +162,7 @@
                             </group>
                         </group>
                     </group>
-                    <group name="state" attrs="{'invisible': [('id','=',False)]}">
```

Comment:
That's exactly the issue. You could technically have a "default" view that would be used for the create view, then make customized form views linked to specific actions.

If the `id` domain didn't work, I think the `created_at` is the next best bet then. What is the issue with the `("id", "=", False)` though?